src/ServiceControl.Audit.Persistence.RavenDb5/Indexes/SagaDetailsIndex.cs

```diff
@@ -52,6 +52,9 @@ into g
                                         .OrderByDescending(x => x.FinishTime)
                                         .ToList()
                                 };
+
+            Index(x => x.SagaType, FieldIndexing.No);
+            Index(x => x.Changes, FieldIndexing.No);
```

Comment:
I think we also need to add

```
Stores.Add(x => x.Changes, FieldStorage.Yes);
```

as described in https://ravendb.net/docs/article-page/5.4/Csharp/indexes/using-analyzers#manipulating-field-indexing-behavior

Reply:
Tests are all green but I'll add a few new ones to make sure, stat tuned

Reply:
@SzymonPobiega we already have tests to check that we return data see https://github.com/Particular/ServiceControl/blob/master/src/ServiceControl.Audit.Persistence.Tests/SagaHistoryTests.cs#L28

I did add one new test to check that we do not index the saga type and changes columns

Reply:
I think the difference might be that if you don't do `FieldStorage.Yes` then when you query by index, it will have to load the documents to return the results. But now that I think about it, that's actually the correct way as the system is write-heavy, not read-heavy. So 👍